traffic_monitor/datareq/datareq.go

```diff
@@ -286,24 +287,56 @@ func WrapAgeErr(errorCount threadsafe.Uint, f func() ([]byte, time.Time, error),
 	}
 }
 
+func accessLogTime(t time.Time) float64 {
+	return float64(t.Unix()) + (float64(t.Nanosecond()) / 1e9)
```

Comment:
Oops, I think I forgot to add a comment I had written here. Can this be simplified to this?
```
float64(t.UnixMilli()) / 1000.0
```